OpenRA.Mods.Common/Traits/Infantry/ScaredyCat.cs

```diff
@@ -82,12 +82,14 @@ public void Damaged(Actor self, AttackInfo e)
 				Panic();
 		}
 
-		public void Attacking(Actor self, Target target, Armament a, Barrel barrel)
+		void INotifyAttack.Attacking(Actor self, Target target, Armament a, Barrel barrel)
```

Comment:
Does anyone know what the purpose of this is?  It was introduced in 11c45e43884f57de6daa2a38543eb7670a0c99e4 but doesn't appear to be used at all.